src/views/components/attendees-list/attendees/title.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Generic: Success Order Page Attendee list - title
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/components/attendees-list/attendees/title.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
```

Comment:
```suggestion
 * @since TBD
```

last couple of adjustments. Otherwise, good to go.